lib/std/fs/test.zig

```diff
@@ -178,6 +178,19 @@ fn testReadLinkAbsolute(target_path: []const u8, symlink_path: []const u8) !void
     try testing.expectEqualStrings(target_path, given);
 }
 
+test "Dir.symLink with relative target that has a / path separator" {
+    var tmp = testing.tmpDir(.{});
+    defer tmp.cleanup();
+
+    try tmp.dir.makePath("a");
+    try tmp.dir.writeFile("a/file", "");
```

Comment:
Most tests here use the `fs.path.sep_str`, so maybe add a comment that the hard-coded path separator here ("/") is intentional here because you want to test Window's handling of the "wrong" separator (which it supports).

Alternatively, all the tests that use `fs.path.sep_str` could instead iterate through the supported separator characters (so on Linux, its just the `/`, but on Windows you'd try both?).  (E.g., I think the test up on L159-164 is the same as this one except for the sep_str?)

Reply:
Good call, ended up going with the 'test both path separators on Windows' route and removed this test since it's now redundant.